### PR TITLE
feat: translate tscofing infos into portuguese/br 

### PR DESCRIPTION
**Pull request by developerfred**
#233
 * - [x] incremental.md
 * - [x] inlineSourceMap.md
 * - [x] inlineSources.md
 * - [x] isolatedModules.md
 * - [x] jsxFactory.md
 * - [x] keyofStringsOnly.md
 * - [x] lib.md
 * - [x] listEmittedFiles.md
 * - [x] listFiles.md
 * - [x] locale.md
 * - [x] mapRoot.md
 * - [x] maxNodeModuleJsDepth.md
 * - [x] module.md
 * - [x] newLine.md
 * - [x] noEmit.md
 * - [x] noEmitHelpers.md
 * - [x] noEmitOnError.md
 * - [x] noErrorTruncation.md
 * - [x] noFallthroughCasesInSwitch.md

### CODE REVIEW

File: packages/tsconfig-reference/copy/pt/options/incremental.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+display: "Incremental"
+oneline: "Salve arquivos .tsbuildinfo para permitir a compilação incremental de projetos"
+---
+
+Diz ao TypeScript para salvar informações sobre o gráfico do projeto da última compilação em arquivos armazenados no disco.
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/incremental.md]:
```suggestion
Diz ao TypeScript para salvar informações sobre o grafo do projeto da última compilação em arquivos armazenados no disco.
```

graph -> grafo nesse contexto

File: packages/tsconfig-reference/copy/pt/options/jsxFactory.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+display: "JSX Factory"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/jsxFactory.md]:
```suggestion
display: "Fábrica JSX"
```
Display deve ser traduzido. É mostrado ao lado da flag em si, não tem problema ser em português

File: packages/tsconfig-reference/copy/pt/options/inlineSourceMap.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+display: "Inline Source Map"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/inlineSourceMap.md]: Traduzir display

File: packages/tsconfig-reference/copy/pt/options/isolatedModules.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+// @showEmit
+// @removeComments
+declare const enum Numbers {
+  Zero = 0,
+  One = 1
+}
+console.log(Numbers.Zero + Numbers.One);
+```
+
+Sem o conhecimento dos valores desses membros, outros transpiladores não podem substituir as referências a `Number`, o que seria um erro de tempo de execução se deixado sozinho (uma vez que não há objeto` Numbers` em tempo de execução).
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/isolatedModules.md]:
```suggestion
Sem o conhecimento dos valores desses membros, outros transpiladores não podem substituir as referências a `Number`, o que seria um erro de tempo de execução se deixado sozinho (uma vez que não há objeto `Numbers` em tempo de execução).
```

File: packages/tsconfig-reference/copy/pt/options/jsxFactory.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+oneline: "Controle a função emitida pelo JSX"
+---
+
+Altera a função chamada em arquivos `.js` ao compilar elementos JSX.
+A alteração mais comum é usar `"h"` ou `"preact.h"` ao invés do padrão `"React.createElement"` se estiver usando `preact`.
+
+Esta opção pode ser usada por arquivo muito semelhante a [Babel's `/** @jsx h */` directive](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#custom).
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/jsxFactory.md]:
```suggestion
Esta opção pode ser usada por arquivo muito semelhante a [diretiva `/** @jsx h */` do Babel](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#custom).
```

File: packages/tsconfig-reference/copy/pt/options/keyofStringsOnly.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
+display: "Keyof Strings Only"
+oneline: "Faça com que keyof retorne apenas strings em vez de strings ou números"
+---
+
+Este sinalizador muda o operador do tipo `keyof` para retornar` string` em vez de `string | number` quando aplicado a um tipo com uma assinatura de índice de string.
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/keyofStringsOnly.md:6]:
```suggestion
Este sinalizador muda o operador do tipo `keyof` para retornar `string` em vez de `string | number` quando aplicado a um tipo com uma assinatura de índice de string.
```

File: packages/tsconfig-reference/copy/pt/options/keyofStringsOnly.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+
+Este sinalizador é usado para ajudar as pessoas a evitar esse comportamento de [before TypeScript 2.9's release](/docs/handbook/release-notes/typescript-2-9.html#support-number-and-symbol-named-properties-with-keyof-and-mapped-types).
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/keyofStringsOnly.md:8]:
```suggestion
Este sinalizador é usado para ajudar as pessoas a evitar esse comportamento de [antes do lançamento do TypeScript 2.9](/docs/handbook/release-notes/typescript-2-9.html#support-number-and-symbol-named-properties-with-keyof-and-mapped-types).
```

File: packages/tsconfig-reference/copy/pt/options/lib.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+- Sua plataforma de tempo de execução fornece certos objetos de API JavaScript (talvez por meio de polyfills), mas ainda não suporta a sintaxe completa de uma determinada versão ECMAScript
+- Você tem polyfills ou implementações nativas para alguns, mas não todos, de uma versão ECMAScript de nível superior
+
+### Bibliotecas de alto nível
+
+| Name         | Contents                                                                                                                                          |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `ES5`        | Definições básicas para todas as funcionalidades ES3 e ES5                                                                                                |
+| `ES2015`     | APIs adicionais disponíveis no ES2015 (também conhecido como ES6) - `array.find`, `Promise`, `Proxy`, `Symbol`, `Map`, `Set`, `Reflect`, etc.               |
+| `ES6`        | Alias for "ES2015"                                                                                                                                |
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/lib.md:21]:
```suggestion
| `ES6`        | Apelido para "ES2015"                                                                                                                                |
```

File: packages/tsconfig-reference/copy/pt/options/lib.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+| `ES2016`     | APIs adicionais disponíveis no ES2016 - `array.include`, etc.                                                                                       |
+| `ES7`        | Alias for "ES2016"                                                                                                                                |
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/lib.md:23]:
```suggestion
| `ES7`        | Apelido para "ES2016"                                                                                                                                |
```

File: packages/tsconfig-reference/copy/pt/options/lib.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+| `ES2017`     | APIs adicionais disponíveis no ES2017 - `Object.entries`, `Object.values`, `Atomics`, `SharedArrayBuffer`, `date.formatToParts`, typed arrays, etc. |
+| `ES2018`     | APIs adicionais disponíveis no ES2018 - `async` iterables, `promise.finally`, `Intl.PluralRules`, `rexexp.groups`, etc.                             |
+| `ES2019`     | APIs adicionais disponíveis no ES2019 - `array.flat`,` array.flatMap`, `Object.fromEntries`, `string.trimStart`, `string.trimEnd`, etc.             |
+| `ES2020`     | APIs adicionais disponíveis no ES2020 - `string.matchAll`, etc.                                                                                     |
+| `ESNext`     | APIs adicionais disponíveis no ESNext - Isso muda conforme a especificação do JavaScript evolui                                                       |
+| `DOM`        | [DOM](https://developer.mozilla.org/docs/Glossary/DOM) definitions - `window`, `document`, etc.                                                   |
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/lib.md:29]:
```suggestion
| `DOM`        | Definições de [DOM](https://developer.mozilla.org/docs/Glossary/DOM) - `window`, `document`, etc.                                                   |
```

File: packages/tsconfig-reference/copy/pt/options/lib.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+| `WebWorker`  | APIs disponíveis em [Web Worker](https://developer.mozilla.org/docs/Web/API/Web_Workers_API/Using_web_workers) contexts                              |
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/lib.md:30]:
```suggestion
| `WebWorker`  | APIs disponíveis em contextos de [Web Worker](https://developer.mozilla.org/docs/Web/API/Web_Workers_API/Using_web_workers) |
```

File: packages/tsconfig-reference/copy/pt/options/lib.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+| `ScriptHost` | APIs para o [Windows Script Hosting System](https://wikipedia.org/wiki/Windows_Script_Host)                                                      |
+
+### Individual library components
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/lib.md:33]:
```suggestion
### Componentes individuais de bibliotecas
```

File: packages/tsconfig-reference/copy/pt/options/lib.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+
+| Name                      |
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/lib.md:35]:
```suggestion
| Nome                      |
```

File: packages/tsconfig-reference/copy/pt/options/lib.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+| `ES2019.Symbol`           |
+| `ES2020.Full`             |
+| `ES2020.String`           |
+| `ES2020.Symbol.wellknown` |
+| `ESNext.AsyncIterable`    |
+| `ESNext.Array`            |
+| `ESNext.Intl`             |
+| `ESNext.Symbol`           |
+
+Esta lista pode estar desatualizada, você pode ver a lista completa no [TypeScript source code](https://github.com/microsoft/TypeScript/tree/master/lib).
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/lib.md:69]:
```suggestion
Esta lista pode estar desatualizada, você pode ver a lista completa no [código fonte do TypeScript](https://github.com/microsoft/TypeScript/tree/master/lib).
```

File: packages/tsconfig-reference/copy/pt/options/listEmittedFiles.md [hunk 15]
```diff
@@ -0,0 +1,42 @@
...
+}
+```
+
+Ecoaria caminhos como:
+
+```
+$ npm run tsc
+
+path/to/example/index.js
+path/to/example/index.d.ts
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/listEmittedFiles.md:39]:
```suggestion
caminho/para/exemplo/index.js
caminho/para/exemplo/index.d.ts
```

File: packages/tsconfig-reference/copy/pt/options/listFiles.md [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+Ecoaria caminhos como::
+
+```
+$ npm run tsc
+path/to/example/node_modules/typescript/lib/lib.d.ts
+path/to/example/node_modules/typescript/lib/lib.es5.d.ts
+path/to/example/node_modules/typescript/lib/lib.dom.d.ts
+path/to/example/node_modules/typescript/lib/lib.webworker.importscripts.d.ts
+path/to/example/node_modules/typescript/lib/lib.scripthost.d.ts
+path/to/example/index.ts
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/listFiles.md:37]:
```suggestion
caminho/para/exemplo/node_modules/typescript/lib/lib.d.ts
caminho/para/exemplo/node_modules/typescript/lib/lib.es5.d.ts
caminho/para/exemplo/node_modules/typescript/lib/lib.dom.d.ts
caminho/para/exemplo/node_modules/typescript/lib/lib.webworker.importscripts.d.ts
caminho/para/exemplo/node_modules/typescript/lib/lib.scripthost.d.ts
caminho/para/exemplo/index.ts
```

File: packages/tsconfig-reference/copy/pt/options/locale.md [hunk 17]
```diff
@@ -0,0 +1,26 @@
+---
+display: "Locale"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/locale.md:2]:
```suggestion
display: "Idioma"
```

File: packages/tsconfig-reference/copy/pt/options/locale.md [hunk 18]
```diff
@@ -0,0 +1,26 @@
...
+oneline: "Defina o idioma da saída tsc"
+---
+
+TypeScript suporta uma série de linguagens, no entanto, isso não pode ser definido por meio do `tsconfig.json`, apenas
+através do ambiente host ou da bandeira CLI `--locale`. Por exemplo. `tsc --locale ru`.
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/locale.md:7]:
```suggestion
através do ambiente host ou da opção de CLI `--locale`. Por exemplo. `tsc --locale ru`.
```

File: packages/tsconfig-reference/copy/pt/options/locale.md [hunk 19]
```diff
@@ -0,0 +1,26 @@
...
+
+### Localidades com suporte a partir de TypeScript 3.7
+
+| Name                  | Locale  |
+| --------------------- | ------- |
+| Brazilian Portuguese  | `pr-BR` |
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/locale.md:13]:
```suggestion
| Brazilian Portuguese  | `pt-BR` |
```
@orta I found a typo on the english docs

File: packages/tsconfig-reference/copy/pt/options/mapRoot.md [hunk 20]
```diff
@@ -0,0 +1,18 @@
+---
+display: "Raiz do mapa"
+oneline: "Defina uma raiz externa para mapas de origem"
+---
+
+Especifique o local onde o depurador deve localizar os arquivos de mapa em vez dos locais gerados.
+Esta string é tratada literalmente dentro do source-map, por exemplo:
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/mapRoot.md:7]:
```suggestion
Esta string é tratada literalmente dentro do mapa de origem, por exemplo:
```

File: packages/tsconfig-reference/copy/pt/options/maxNodeModuleJsDepth.md [hunk 21]
```diff
@@ -0,0 +1,11 @@
+---
+display: "Profundidade JS do módulo de nó máximo"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/maxNodeModuleJsDepth.md:2]:
```suggestion
display: "Profundidade máxima do nó de módulo JS"
```

File: packages/tsconfig-reference/copy/pt/options/noEmitHelpers.md [hunk 22]
```diff
@@ -0,0 +1,38 @@
+---
+display: "No Emit Helpers"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/noEmitHelpers.md:2]:
```suggestion
display: "Não emitir auxiliares"
```

File: packages/tsconfig-reference/copy/pt/options/noEmitHelpers.md [hunk 23]
```diff
@@ -0,0 +1,38 @@
...
+oneline: "Suponha que helpers estejam disponíveis no tempo de execução global"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/noEmitHelpers.md]:
```suggestion
oneline: "Suponha que auxiliares estejam disponíveis no tempo de execução global"
```

File: packages/tsconfig-reference/copy/pt/options/noEmitHelpers.md [hunk 24]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+Em vez de importar auxiliares com [`importHelpers`](#importHelpers), você pode fornecer implementações no escopo global para os auxiliares que você usa e desligar completamente a emissão de funções auxiliares.
+
+Por exemplo, usar esta função `async` no ES5 requer uma função do tipo` await` e una função do tipo `generator` para executar:
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/noEmitHelpers.md]:
```suggestion
Por exemplo, usar esta função `async` no ES5 requer uma função do tipo `await` e uma função do tipo `generator` para executar:
```

File: packages/tsconfig-reference/copy/pt/options/noEmitOnError.md [hunk 25]
```diff
@@ -0,0 +1,8 @@
+---
+display: "No Emit On Error"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/noEmitOnError.md]:
```suggestion
display: "Não emitir em caso de erro"
```

File: packages/tsconfig-reference/copy/pt/options/noErrorTruncation.md [hunk 26]
```diff
@@ -0,0 +1,40 @@
+---
+display: "No Error Truncation"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/noErrorTruncation.md]:
```suggestion
display: "Sem truncamento de erro"
```

File: packages/tsconfig-reference/copy/pt/options/noFallthroughCasesInSwitch.md [hunk 27]
```diff
@@ -0,0 +1,22 @@
+---
+display: "No Fallthrough Cases In Switch"
+oneline: "Relate erros para casos de falha nas instruções switch."
+---
+
+Reportar erros para casos de casesRelate erros para casos de fallthrough em instruções switch.
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/noFallthroughCasesInSwitch.md]:
```suggestion
Reportar erros para casos de fallthrough em instruções switch.
```

**Review by danilofuchs — CHANGES_REQUESTED** [review]
Great work! I noticed some typos and weird translations.
Be sure all `display` fields are correctly translated

**Comment by orta** [comment]
The red is my issue, it can either be rebased now that #930  is merged, or ignored which is totally OK too

**Comment by developerfred** [comment]
@orta done

**Comment by orta** [comment]
hah, OK, looks like the fix did not work like I hoped 👍🏻

File: packages/tsconfig-reference/copy/pt/options/isolatedModules.md [hunk 28]
```diff
@@ -0,0 +1,75 @@
+---
+display: "Isolated Modules"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/isolatedModules.md:2]:
```suggestion
display: "Módulos isolados"
```

File: packages/tsconfig-reference/copy/pt/options/inlineSources.md [hunk 29]
```diff
@@ -0,0 +1,36 @@
+---
+display: "Inline Sources"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/inlineSources.md:2]:
```suggestion
display: "Origens embutidas"
```
Não tenho certeza dessa tradução hehe

File: packages/tsconfig-reference/copy/pt/options/keyofStringsOnly.md [hunk 30]
```diff
@@ -0,0 +1,8 @@
+---
+display: "Keyof Strings Only"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/keyofStringsOnly.md:2]:
```suggestion
display: "Somente strings em keyof"
```

File: packages/tsconfig-reference/copy/pt/options/lib.md [hunk 31]
```diff
@@ -0,0 +1,69 @@
+---
+display: "Lib"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/lib.md:2]:
```suggestion
display: "Biblioteca"
```

File: packages/tsconfig-reference/copy/pt/options/noFallthroughCasesInSwitch.md [hunk 32]
```diff
@@ -0,0 +1,22 @@
+---
+display: "No Fallthrough Cases In Switch"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/noFallthroughCasesInSwitch.md:2]:
```suggestion
display: "Sem casos de Fallthrough em Switch"
```

**Comment by khaosdoctor** [comment]
@developerfred Ping! :) 

Can you please check into @danilofuchs' corrections so we can merge it?

Thanks!

**Comment by khaosdoctor** [comment]
@orta can you please close this PR too? I created #997 to replace it

**Pull request merged**